### ISSUE TITLE
openshift-install ctrl-c does not quit the process

### ISSUE THREAD

**Issue by rphillips** [openshift/installer#366]
When running openshift-install, the application does not exit when ctrl-c is pressed at the prompts.

**Comment by wking** [comment]
Yeah, this is annoying.  Looks like we need to trace [this][1] through to see if it's something we can notice and die on.

/assign

[1]: https://github.com/AlecAivazis/survey/blob/f30c5d1830c892f533140f29a1de89141dc217f5/editor.go#L102-L103
